public/pages/CreateRollup/utils/constants.ts

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+export const EMPTY_ROLLUP = JSON.stringify({
+  rollup: {
+    continuous: false,
+    description: "",
+    dimensions: [
+      {
+        date_histogram: {
+          source_field: "",
+          fixed_interval: "1h",
+          timezone: "UTC",
+        },
+      },
+    ],
+    enabled: true,
+    metrics: [],
+    page_size: 1000,
+    roles: [],
+    schedule: {
+      interval: {
+        start_time: 234802,
```

Comment:
Changed these values to match the initial value I set in `CreateRollup` page, and removed `start_time` in `EMPTY_ROLLUP` since this is not needed when creating a new rollup job. 

This `EMPTY_ROLLUP` is used in `CreateRollup` as initial JSON string, and in `EditRollup` and `RollupDetails` so that it is easier to set values after calling `getRollup`. 

Changed initial value of `rollupJSON` to `""` in `RollupDetails` page since response can be directly set as rollupJSON, but kept `EMPTY_ROLLUP` as initial value of`rollupJSON` in both`CreateRollup` and`EditRollup` page to make sure the PUT rollup API is working correctly.